File: app/src/main/java/com/riders/thelab/data/local/bean/TimeOut.java
package com.riders.thelab.data.local.bean;

public enum TimeOut {

    TIME_OUT_READ(60),
    TIME_OUT_CONNECTION(60);

    private final int value;

    TimeOut(int value) {
        this.value = value;
    }

    public int getValue() {
        return value;
    }
}

File: core/data/src/main/java/com/riders/thelab/core/data/local/bean/TimeOut.kt
package com.riders.thelab.core.data.local.bean

enum class TimeOut(val value: Int) {
    TIME_OUT_READ(60),
    TIME_OUT_CONNECTION(60);

    companion object {
        fun getValue(value: TimeOut): Int = entries.first { it.name == value.name }.value
    }
}